omero/developers/Web/PublicData.txt

```diff
@@ -29,50 +29,42 @@ Here is how to go about setting this up on your OMERO.web install:
 
         $ bin/omero config set omero.web.public.password '<password>'
 
--   Set a URL filter for which the OMERO.web public user is allowed to 
-    navigate.
-    Default: '^/(?!webadmin)' (Python reqular expression). You probably do not
-    want the whole webclient UI to be publicly visible (although you could do
-    this).
+-   Set the ``omero.web.public.url_filter``. This filter is a
+    regex expression that will only allow matching urls to be accessed
```

Comment:
This is the same as on the other branch at the moment, so I'm inclined to leave it so we don't end up out of sync.